XmlFormatter/src/Settings/Provider/DataStructure/SerializableSetting.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace XmlFormatter.src.Settings.Provider.DataStructure
+{
+    /// <summary>
+    /// This class is a data container to for the xml saving
+    /// </summary>
+    public class SerializableSetting
+    {
+        /// <summary>
+        /// The key/name of the setting
+        /// </summary>
+        private string name;
+
+        /// <summary>
+        /// Public access to hey key/name of this setting
+        /// </summary>
+        public string Name
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this an auto-implemented property and remove its backing field.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5052706)